pkg/reconciler/knativeserving/ingress/kourier.go

```diff
@@ -140,3 +138,10 @@ func configureBootstrapConfigMap(instance *v1beta1.KnativeServing) mf.Transforme
 		return nil
 	}
 }
+
+func hasProviderLabel(u *unstructured.Unstructured) bool {
```

Comment:
We can drop `hasProviderLabel` in Kourier as well so please feel free to drop it. Or I will drop them in the separate PR.